src/main/java/lotto/LottoResultManager.java

```diff
@@ -1,44 +1,53 @@
 package lotto;
 
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import lotto.domain.LottoNumber;
 import lotto.domain.LottoPrice;
 import lotto.domain.LottoTicket;
 
 public class LottoResultManager {
 
-    private Map<Integer, Integer> lottoResult = new HashMap<>();
+    private final Map<LottoPrice, Integer> lottoResult;
 
     public LottoResultManager() {
-        lottoResult.put(3, 0);
-        lottoResult.put(4, 0);
-        lottoResult.put(5, 0);
-        lottoResult.put(6, 0);
+        this.lottoResult = new LinkedHashMap<>();
+        for (LottoPrice price : LottoPrice.values()) {
+            lottoResult.put(price, 0);
+        }
+
     }
 
-    public Map<Integer, Integer> calculateLottoResult(List<LottoTicket> tickets,
-            List<Integer> winningNumbers) {
+    public Map<LottoPrice, Integer> calculateLottoResult(List<LottoTicket> tickets,
+        List<Integer> winningNumbers, int bonusNumber) {
```

Comment:
```suggestion
    public Map<LottoPrice, Integer> calculateLottoResult(List<LottoTicket> tickets,
        WinLotto winLotto) {
```

LottoResultManager 는 통계만을 내는 클래스로 보입니다.
우승 책임을 맡길 수 있는 객체를 도출해 보면 좋을 것 같습니다! 

추가로 winningNumbers가 정수 리스트이기 때문에 LottoTicket과는 다르게 값 검증이 제대로 이뤄지지 않을 수 있습니다.